### DISCUSSION TITLE
How does one create a non-unique index?

### DISCUSSION THREAD

**Discussion by aaronshaf** [prisma/prisma#4621] [Q&A] [answered]
I have two fields that I want to index, but it shouldn't be a unique index. I am using PostgreSQL. How does one specify this in `schema.prisma`?

**Comment by aaronshaf** [comment, marked as answer]
Found it: https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-an-index

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
